src/control/C4Control.h

```diff
@@ -246,29 +241,28 @@ class C4ControlPlayerMouse : public C4ControlPacket // sync
 		CPM_Drop = 0x02
 	};
 
-	C4ControlPlayerMouse() : action(CPM_NoAction), player(NO_OWNER), target_obj(0), drag_obj(0), old_obj(0) {}
+	C4ControlPlayerMouse() : player(NO_OWNER) {}
 	static C4ControlPlayerMouse *Hover(const C4Player *player, const C4Object *target, const C4Object *old_target, const C4Object *drag = nullptr);
 	static C4ControlPlayerMouse *DragDrop(const C4Player *player, const C4Object *target, const C4Object *drag);
 
 private:
-	int32_t action;
+	int32_t action{CPM_NoAction};
 	int32_t player;
```

Comment:
I noticed that too. I don't know why it was left out (a few other places as well).
I think there's a few other tweaks to be made to my .clang-tidy config file. Maybe enabling `readability-redundant-member-init` will help.
I ran the same script again: it is still finding things to change. I'll upload the new commits after it finishes and I review the changes myself.